### PR TITLE
Fixed richcombo focus issue in Chrome

### PR DESCRIPTION
**Pull request by DmitryKrekota**
I have bug with richcombo focus in Chrome. CKEditor richcombo breaks page because of using focus.
Video recording: https://drive.google.com/file/d/0Bxtv10NlSOEbWXNJbkNFVGQyUG8/view
My pull request fixes this Chrome issue.

### CODE REVIEW

**Comment by f1ames** [comment]
Thanks for the PR!

Looks like very specific case. It may be hard to reproduce and test because it somehow depends on custom code outside the editor. If you could provide a step-by-step instruction on how to reproduce the issue it will be easier for us to analyze and understand where exactly the problem is and what the best solution should be.
Optionally you could also report a bug on [our bugtracker](http://dev.ckeditor.com/) describing steps how to reproduce this issue.

Keep in mind that PR should also contain unit and manual tests (or only manual, if the issue can't be really tested with unit tests).

Thanks again for contribution.

**Comment by DmitryKrekota** [comment]
I created simple demo of this issue: https://drive.google.com/file/d/0Bxtv10NlSOEbdWlsTDZZanBTWjg/view?usp=sharing
How to reproduce:
https://drive.google.com/file/d/0Bxtv10NlSOEbVE94Tkl6dzJSeHM/view

**Comment by f1ames** [comment]
I investigated this issue a little bit further and it seems that this behavior is not the fault of CKEditor but more of how browsers handles specific elements/structure. I modified and simplified your example (which was very helpful by the way) and tested it on Chrome and FF.

It was easier to record a video so here is the link with simplified example and video (along with _video_description.txt_ where I explained what's happening in the video): https://drive.google.com/folderview?id=0B2knXdyRNxzPdFAzaGNwNlMwN1U&usp=sharing.

To sum up the issue (based on provided sample which somehow reflects your application): it seems that the right column even though it's content should not take more space than the window height actually does so window can be actually scrolled - in some specific way (see video I uploaded, it's easy to reproduce on FF even without CKEditor) and adding some additional styling (like _overflow_) can solve this issue.

Hope it helps!

**Comment by DmitryKrekota** [comment]
Thanks for your answer.
